VideoLocker/res/layout/row_course_list.xml

```diff
@@ -2,35 +2,37 @@
 <FrameLayout xmlns:android="http://schemas.android.com/apk/res/android"
     xmlns:tools="http://schemas.android.com/tools"
     android:layout_width="match_parent"
-    android:layout_height="wrap_content"
+    android:layout_height="@dimen/course_card_height"
+    android:descendantFocusability="blocksDescendants"
     android:paddingLeft="@dimen/widget_margin"
-    android:paddingRight="@dimen/widget_margin"
-    android:paddingBottom="@dimen/widget_margin"
-    android:descendantFocusability="blocksDescendants">
+    android:paddingRight="@dimen/widget_margin">
 
-    <RelativeLayout
+    <FrameLayout
         android:layout_width="match_parent"
-        android:layout_height="@dimen/course_card_height"
-        android:background="@color/white">
+        android:layout_height="match_parent"
+        android:background="@color/white"
+        android:duplicateParentState="true"
```

Comment:
Yes, correct, because it's the parent element that receives the selected state.

Ideally, I would rather set the onclick listener on the correct view so that this is not needed (which would also fix the problem that the ripple sometimes appears in the wrong place). But sadly, that would require updating the base class of these adapters which is a lot more refactoring than I was trying to do at this time.